.circleci/config.yml

```diff
@@ -89,6 +90,7 @@ jobs:
       - run:
           name: Tests
           command: |
+            flake8 --count jupyter_book
```

Comment:
One thought -- do we want this as just a count of errors or do we want a separate test ? The reason I ask is that if we want to ensure the codebase stays flake8 compliant, it's harder to remember to go check the circle logs than to just look for a quick pass / fail !

Reply:
It should fail if the count > 0. The reason is that an exit code of 0 means that it was successful. Maybe it's not so obvious? But it's what I've used in other repositories.

Reply:
Ah, I've never used it that way ! Very cool -- thanks for clarifying :smile_cat: 

I guess my question _then_ is if we should have the jupyter-book test fail with flake8 errors (rather than linting being its own test) since we might spin a few more cycles of Circle, as linting errors will need to be resolved before we can see functionality errors :sweat_smile: But I can really see it going either way !

Reply:
You mean the linting error should be thrown after the jupyter-book tests are run? I can change it to be that way if you prefer :) No strong preference either way. But with this approach, you can make a stronger case to make contributors conform to pep8 rather than waiting until the last commit to fix pep8. But yeah, both are fine by me !